Project.toml

```diff
@@ -0,0 +1,21 @@
+name = "MusicProcessing"
+uuid = "f57c4921-e30c-5f49-b073-3f2f2ada663e"
+repo = "https://github.com/JuliaMusic/MusicProcessing.jl.git"
+version = "1.8.1"
+
+[deps]
+DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+SampledSignals = "bd7594eb-a658-542f-9e75-4c4d8908c167"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
```

Comment:
O, forgot to say this here. In the `compat` entry we need an entry for every package used, to allow for registration in the General registry (you can simply use the versions you tested this with in Jupyter)